### PR TITLE
added rspec cases for events_controller.rb

### PR DESCRIPTION
**Pull request by neilgautam24-tamu**

### CODE REVIEW

**Comment by XuMingSheng** [comment]
Put all the files for testing in `spec/fixtures/files`, and change the path in your code accordingly.

**Comment by neilgautam24-tamu** [comment]
@XuMingSheng I have done the mentioned changes.

**Comment by XuMingSheng** [comment]
Put the file for testing in `spec/fixtures/files`.

File: public/uploads/event/event_avatar/3/some-dog.jpg
Comment (XuMingSheng) [inline public/uploads/event/event_avatar/3/some-dog.jpg:1]: Here

File: public/uploads/event/event_box_office/3/Book1.xlsx
Comment (XuMingSheng) [inline public/uploads/event/event_box_office/3/Book1.xlsx:1]: and here

File: app/controllers/guests_controller.rb
Comment (XuMingSheng) [inline app/controllers/guests_controller.rb:1]: remove comment of old code

File: app/controllers/seats_controller.rb
Comment (XuMingSheng) [inline app/controllers/seats_controller.rb:1]: remove comment of old code

File: app/models/seat.rb
Comment (XuMingSheng) [inline app/models/seat.rb:1]: remove comment of old code

File: spec/controllers/events_controller_spec.rb
Comment (XuMingSheng) [inline spec/controllers/events_controller_spec.rb:1]: remove comment of old code

Comment (neilgautam24-tamu) [reply spec/controllers/events_controller_spec.rb:1]: Removed

File: app/controllers/guests_controller.rb
Comment (neilgautam24-tamu) [reply app/controllers/guests_controller.rb:1]: Removed that snippet

File: spec/controllers/events_controller_spec.rb
Comment (XuMingSheng) [reply spec/controllers/events_controller_spec.rb:1]: Some of them are still there.

File: spec/controllers/events_controller_spec.rb [hunk 1]
```diff
@@ -15,17 +15,57 @@
       expect(response).to be_successful
     end
   end
-  describe 'GET #index' do
-    it 'returns a success response' do
+
+  # describe "GET #index" do
```
Comment (XuMingSheng) [inline spec/controllers/events_controller_spec.rb:19]: here

File: spec/controllers/events_controller_spec.rb [hunk 2]
```diff
@@ -40,8 +80,27 @@
     end
   end
 
-  describe 'GET #new' do
-    it 'returns a success response' do
+  # describe "GET #show" do
```
Comment (XuMingSheng) [inline spec/controllers/events_controller_spec.rb:83]: here

File: app/models/seat.rb
Comment (XuMingSheng) [reply app/models/seat.rb:1]: Some of them are still there.

File: app/models/seat.rb [hunk 3]
```diff
@@ -1,13 +1,35 @@
-# frozen_string_literal: true
+# class Seat < ApplicationRecord
```
Comment (XuMingSheng) [inline app/models/seat.rb:1]: Here

**Review by XuMingSheng — CHANGES_REQUESTED** [review]
1. Some of the comments are remained to be removed
2. Some other rspecs (besides events_controller.rb) are not passing in this PR, which are passing in dev branch. Should find the cause and modify this PR to let all rspec pass.

File: features/manage_seat_level.feature [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  Given I am on the events page
+  When I click on the Event
+  And I click on the "Add New Seat" button
+  And I fill in "Category" with "Guest List"
+  And I fill in "Section" with "VIP"
+  And I fill in "Total Count" with "40"
+  And I click on the "Create Seat" button
+  Then I should see "Guest List" in the Manage Seating Levels list
+
+Scenario: Edit event details
```
Comment (XuMingSheng) [inline features/manage_seat_level.feature:13]: Why is this scenario be here? Shouldn't it be in a separate feature such as `events.feature`?

**Review by XuMingSheng — APPROVED** [review]

**Pull request merged**